packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -788,7 +946,7 @@ const buildPolicyResource = (resourceName: string, path: string | null) => {
 const templateSchemaFilter = authConfig => {
   const authIncludesCognito = getAuthTypes(authConfig).includes('AMAZON_COGNITO_USER_POOLS');
   return (templateOption: ListChoiceOptions): boolean =>
-    authIncludesCognito || templateOption.value !== 'single-object-auth-schema.graphql';
+    authIncludesCognito || templateOption.name !== 'Objects with fine-grained access control (e.g., a project management app with owner-based authorization)';
```

Comment:
using 'value' will require expanding the conditional statement. Since the name is same for both the versions, I think it would keep the condition tidy.